sunpy/net/dataretriever/sources/tests/test_goes_ud.py

```diff
@@ -39,11 +36,11 @@ def test_get_url_for_time_range(LCClient, timerange, url_start, url_end):
 @pytest.mark.remote_data
 @pytest.mark.parametrize("timerange, url_start, url_end",
                          [(TimeRange('1999/01/10', '1999/01/20'),
-                           'https://umbra.nascom.nasa.gov/goes/fits/1999/go10990110.fits',
+                           'https://umbra.nascom.nasa.gov/goes/fits/1999/go0819990115.fits',
                            'https://umbra.nascom.nasa.gov/goes/fits/1999/go1019990120.fits')])
 def test_get_overlap_urls(LCClient, timerange, url_start, url_end):
-    urls = LCClient._get_url_for_timerange(timerange)
-    assert len(urls) == 9
+    urls, urlmeta = LCClient._get_url_for_timerange(timerange)
+    assert len(urls) == 28
```

Comment:
So our old requests get more results now?

Reply:
This was because if we don't define specify level in query of `lyra`, data for level 1,2,3 are returned. No level is favoured or defaulted over other.